symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigLoaderTest.java

```diff
@@ -84,4 +85,16 @@ public void loadFromClasspathNotFoundTest() throws BdkConfigException {
         });
         assertEquals(exception.getMessage(), "Config file is not found");
     }
+
+    @Test
+    public void loadClientGlobalConfig() throws BdkConfigException {
+        BdkConfig config = BdkConfigLoader.loadFromClasspath("/config/config_client_global.yaml");
+        assertEquals(config.getPod().getHost(), "devx1.symphony.com");
+        assertEquals(config.getPod().getPort(), 8443);
+        assertEquals(config.getPod().getContext(), "context");
+        assertEquals(config.getScheme(), "https");
+        assertEquals(config.getAgent().getPort(), 443);
+        assertEquals(config.getAgent().getFormattedContext(), "/context");
+        assertEquals(config.getKeyManager().getFormattedContext(), "/diff-context");
```

Comment:
do we test all the possible cases here? (like if the pod's host can be overriden?)
(ok I'm being picky but maybe it is worth it for the SDK)

Reply:
I will add more test case here.